src/content/time-tracking.mdx

```diff
@@ -0,0 +1,65 @@
+
+
+## Tracking good manners
+
+### Adding a description
+As mentioned a few times earlier, it’s an expected practice to add descriptions to your tracked positions. **Remember that you don’t need to describe them in details. Try to use a short definition of described task.** E.g. daily, planning, refinement (for meetings), Accessibility/A11y, TypeScript generics, Unit tests (for learning), a copy-pasted name of your dev task. The main goal of a description is to easy&fast define what it is, to easy&fast inform a PM about it.
+
+### Regularity - keep your PM always informed
+It’s not obvious but **your timesheet should always be up to date**. It’s not good to fill a timesheed at the end of a day or a week, because it’s impossible to fill it accurately this way and it’s hard to prevent extra hours you spend on a task. **The best and expected way of tracking time is to keep a timer always running, when you’re working.**
+It’s not only for PMs, but for you as developer too, because when it will be visible in a timesheet, that you’re working on an important task now, then nobody will disturb you :)
+
+### Overtime
+It’s important to **control your timesheet by yourself to prevent extra hours.** It means you should control your estimated vs spent time on a task, your break time (don’t go beyond a limit, but respect a daily rest) and your weekly total hours. Remember to control it always, even if it’s a release time or something.
+If you know you won’t finish a task in an estimated time, absolutely inform your PM about it.
+
+### Weekly submitting
+Your timesheet should be weekly verified by your PM, so if the week (or something a month) is over, then submit your timesheet for approval. You can do it until EOD.
+
+## Free time
+
+### Break
+It couldn’t be unmentioned! Our break should be also tracked. Use general tracking rules for this too, but remember that break is free from description. It’s your time :)
+
+### Holidays
+Holidays should be also tracked. It doesn’t mean you should start your timer when you start a holidays and stop it when you’re back. Just add 8hours per each holiday day in a project Public holiday/Sickness/Vacation (relatively to holidays type).
+
+
+## Tracking fails
+Don’t panic! Probably sometimes tracking fails will occur:
+
+### You forgot to start your timer when you started a task
+Of course try to avoid these situations, but if it occurs then just update a time spent with a proper amount. If it’s a project task, then you can use a synchronization feature in JIRA (Avatar -> Synchronize timers to issues -> filter by expected day -> manually do a connection with a „>>” button), to update JIRA timer with manually updated Harvest.  Please be careful with it, because if it’s a next part of time spent, it could be replaced instead of totaled up.
```

Comment:
```suggestion
Of course, try to avoid these situations, but if it does happen, just update the time spent with a proper amount.  If it’s a project task, then you can use the synchronization feature in JIRA (Avatar -> Synchronize timers to issues -> filter by expected day -> manually do a connection with a „>>” button) to update the timer with the Harvest's value.  Please be careful with it! It’s a next part of time spent so it could be replaced instead of totaled up.
```